README.md

```diff
@@ -0,0 +1,7 @@
+<img width="226" alt="image" src="https://user-images.githubusercontent.com/26731448/188519846-a027234f-8008-40cb-8b22-87a233650de0.png">
+
+## Sub strings
+
+A challenge on [Odin's project](https://www.theodinproject.com/lessons/ruby-sub-strings)
```

Comment:
English tip: It is the project named Odin, not the project that belongs to Odin :P
```suggestion
A challenge on [Odin Project](https://www.theodinproject.com/lessons/ruby-sub-strings)
```